### DISCUSSION TITLE
SunbirdEd- As a creator unable to see all my contents (with relevant status) under DRAFT, SUBMITTED FOR REVIEW and PUBLISHED section of Workspeace

### DISCUSSION THREAD

**Discussion by devendra-tarento** [Sunbird-Ed/Community#329] [Bugs] [answered]
As contributor I have created some question sets and submitted them for review and also published few of them using relevant api in postman. 

I am able to see all of them (With status as draft, live and review) in **ALL MY CONTENT** section of **Workspace**:

<img width="1674" alt="Creator-All my content " src="https://user-images.githubusercontent.com/114464145/210199894-21a1777a-8c28-4884-936f-c1fd1dabba2d.png">

But I don't see them (with status as draft) in **Draft** section of **Workspace**:

<img width="1680" alt="Creator-Draft" src="https://user-images.githubusercontent.com/114464145/210200486-7dfbb813-16ce-4efb-8b9b-88363564f9c6.png">

I don't see them (with status as review) in **Submitted For Review** section of **Workspace**:

<img width="1680" alt="Creator-Submitted  For review" src="https://user-images.githubusercontent.com/114464145/210200549-b9ba40f1-aa0e-40af-ad8e-68d46a6e96cb.png">

I don't see them (with status as live) in **Published** section of **Workspace**:

<img width="1677" alt="Creator- Published" src="https://user-images.githubusercontent.com/114464145/210200613-8f867695-ba7c-49a6-bb46-d186b9703ad4.png">

**Comment by Krishnaj20** [comment, marked as answer]
@devendra-tarento - The creation workflows are powered by coKreat. Please raise this issue in coKreat forums. Link is: https://github.com/Sunbird-coKreat/Community/discussions

@luckynvdu
